tests/data/Alternates-Regular.ufo/lib.plist

```diff
@@ -0,0 +1,596 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>com.github.googlei18n.ufo2ft.filters</key>
+    <array>
+      <dict>
+        <key>name</key>
+        <string>eraseOpenCorners</string>
+        <key>namespace</key>
+        <string>glyphsLib.filters</string>
+        <key>pre</key>
+        <true/>
+      </dict>
+    </array>
+    <key>com.schriftgestaltung.Glyphs.glyphUserData.ka-oriya.below</key>
```

Comment:
... same with all the `com.schriftgestaltung.*` private glyphsLib stuff that ufo2ft doesn't care about